packages/docusaurus-plugin-content-blog/src/__tests__/__fixtures__/website/blog/blog-with-links.mdx

```diff
@@ -0,0 +1,22 @@
+import dino from "../static/img/docusaurus.png";
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
```

Comment:
ES imports are supposed to come after front matter

Reply:
Order of imports are now correct. I also moved the front matter to the beginning of the file. Misplacing front matter seems to be the reason why tests failed.